lib/search.js

```diff
@@ -8,29 +8,83 @@ const octokit = require('@octokit/rest')({
   }
 })
 
-// GitHub search parameters for the Node.js org
-var sort = 'updated'
-var order = 'desc'
+/// GitHub search parameters for the Node.js org
+const SORT = 'updated'
+const ORDER = 'desc'
+// default params for octokit
+const PER_PAGE = 30
+const PAGE = 1 // page_number is 1-based index
+
+// API does not allow more than 1000 results -> tested via "feeling-lucky"
```

Comment:
So is it possible to change the starting page (PAGE)? It doesn't seem like we're doing that, which would be a vital part of full randomization right? 🤔 

Perhaps I'm just not fully understanding what this code is doing.

Reply:
These are the constants and defaults referred in `getSearchParams` function. Any calls made to `octokit.search.issues` should use `getSearchParams` function inorder to properly generate search-params.

now, although these are defaults, one can override them by passing their search param values.
perhaps, I should use `Object.assign` 😞

Reply:
> So is it possible to change the starting page (PAGE)?

Yes, we are doing it in second call to issues search.